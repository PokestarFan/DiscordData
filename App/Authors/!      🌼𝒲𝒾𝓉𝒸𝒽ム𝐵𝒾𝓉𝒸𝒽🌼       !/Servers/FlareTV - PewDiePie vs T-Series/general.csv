ID,Content Length,Content
558465768443084801,3,uwu
558466578048483330,9,K P O P ?
558466631223738377,20,I have been summoned
558466708244004892,7,Damn it
558466820517003266,41,*crawls back in the hellhole I came from*
558466868529070090,3,uwu
558467003334131749,17,I already do that
558467162398785566,9,I am hell
558467342191820800,35,<@558457743761997825> no_ I AM hell
558467663756525589,2,Ew
558467733851865098,28,Thats nastier than egg salad
558467790739079179,4,Wait
558467948969197589,60,Why do we have so many people named after food in this sever
558468046264729640,8,Probably
558468092653469697,4,Dog?
558468118738108417,14,Dog isn’t food
558468196202577920,19,Crab is also a good
558468258018230283,17,Food_ I mean food
558468323696836620,27,<@!380928993144340482> SAME
558468516924227595,50,<@!380928993144340482> I’ve never had crAwfish ;-;
558468576000868382,17,Hi bored_ I’m dad
558468680195768320,37,<@!380928993144340482> I just havrnts
558468757333213211,16,Stop flexing aaa
558468805743869998,39,<@!380928993144340482> I will then lmao
558469005073973268,39,<@!380928993144340482> I don’t think so
558469086317772830,30,*crawls out of hellhole* kPoP?
558469157608489020,27,<@558457743761997825> what?
558469381953159168,32,<@!380928993144340482> oooh okay
558469529177686017,34,Imma block eclipse if Chelsea does
558469548789989386,6,Oh rip
558469642855907328,34,Damn I needed a reason to be petty
558469747008733194,4,Yeah
558469812834140181,3,Ban
558470043776581632,3,Rip
558470153851895839,3,Yes
558470374392725517,11,Hit or miss
558470421343633408,27,I guess they never miss huh
558470443762450444,19,You got a boyfriend
558470476377096200,24,I bet he doesn’t kiss ya
558470519138287617,4,Muga
558470548598816778,7,PRIVATE
558470559483035668,7,YOU HOE
558470610825510922,32,Stop making me sound like a thot
558470708687011848,13,*no it’s not*
558470742623125517,2,NO
558470758737641485,4,Lmao
558470858683711488,25,Excuse me_ I’m a guy lmao
558471124900380693,17,Egg is not a girl
558471316299186186,2,he
558471329993719819,8,I’m a he
558471663167995914,33,<@!380928993144340482> crab is 10
558472219748204548,21,There’s a 10 year old
558473347428515841,0,
558778138558070795,4,Shit
558778153582198784,4,Fucc
558778191238660096,5,Bitsh
558778226692849664,3,Hoe
558798908881633290,3,uwu
558810331376386072,20,Aaaa I wanna join vc
558810365690118167,21,But I’m in another rn
558810390570729483,14,I can’t choose
558811900645736468,15,can i be banned
558811907855745024,3,pls
558812046381154305,6,i cant
558813039030304799,0,
558813202545377312,22,<@!176921245940580354>
558814011878014976,73,my milkshake brings all the boys to yard_ damn right is better than yours
558815672562483210,7,oh okay
558817008083402763,3,hoe
558818391738286121,25,family friendly porn_ wtf
558818749264691268,11,move me too
558818784387923974,20,wow i didnt even ask
558991746395013121,3,Wot
558992002637627393,2,Oh
558992235622825994,7,Oh nice
558992407476043795,3,oof
558992649881911316,20,Nothing personal kid
558992830853414964,19,Kid is not a virgin
558992953888997399,2,Oh
558993192406482954,8,You suck
558993253920276480,34,Morning Bart <@339845222181896193>
558993353266429973,26,<@!285835383374348289> oOf
558996094672175104,3,Oof
558996235776950283,5,Dunno
