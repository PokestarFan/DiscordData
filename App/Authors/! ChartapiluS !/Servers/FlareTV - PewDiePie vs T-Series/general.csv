ID,Content Length,Content
558996210283839498,37,<@544111003894415362> hee anata dare?
558996232836612126,3,wa*
