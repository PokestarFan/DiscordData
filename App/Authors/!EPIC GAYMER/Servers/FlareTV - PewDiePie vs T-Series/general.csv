ID,Content Length,Content
559098697057566771,2,ok
