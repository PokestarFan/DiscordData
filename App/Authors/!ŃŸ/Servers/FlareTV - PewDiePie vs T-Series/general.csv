ID,Content Length,Content
558443386189905920,3,Ooh
558443483380318219,6,Heller
558443617589526528,3,Oof
558444660264009768,2,Of
558444760461737995,4,Hmmm
558444778887577636,1,üëÄ
558445276545679390,3,Oof
558445305276661771,35,<:kermitragegun:492896354440314880>
558445339871412224,26,<:True:529561303103307776>
558445399187128322,18,Wow no staff is on
558445419177181186,8,Love hat
558445539109371934,3,Wut
558446949695750144,3,Oof
558453776697262127,3,Oof
558453854186766336,3,üëÄüëÄüëÄ
558453865859776512,33,<a:peekstitch:541444790349332510>
558453890891251723,6,Try it
558453900869500938,2,Hi
558453942443573248,4,Good
558454001616814094,4,Cool
558454328814338069,27,<@!518924074198761482> what
558454396514467840,2,Ok
558454542132445214,52,Then I‚Äôll call you ?mute <@!518924074198761482> 1200
558454720067272705,42,<@!341648742685999106> who are you subbing
558454816389464095,8,Stopping
558454827261362196,7,STOOPID
558454869334294578,27,<@!518924074198761482> Baku
558454979745284106,4,Baku
558455014302023731,5,Oh ok
558455044047896609,37,I didn‚Äôt know if this was Baku or not
558455092706017313,3,Lol
558455215234220032,11,That‚Äôs tuff
558455261925212160,3,Yes
558455297337851916,6,That‚Äôs
558455301905580052,4,Tuff
558455311091105792,5,Bitch
558455529534521356,3,Lol
558455689324920852,100,<a:monkaShoot:444921483802705920> <:DeadChat:524896170787733514> <:kermitragegun:492896354440314880>
558456044083347457,3,Oof
558456198794313730,3,Ooh
558456540508717057,3,Oof
558456690568331274,3,Oof
558457142013853716,4,Reee
558457234896584704,2,Oo
558457247521570816,100,<:kermitragegun:492896354440314880> <:MonkaGun:532226138110689280> <a:monkaShoot:444921483802705920>
558457561372819456,30,<:DeadChat:524896170787733514>
558457883520663570,4,No u
558457922393341974,0,
558458090895310848,28,<@339486831991062529> okokok
558458602881417226,3,Oof
558458700365430824,34,Well aren‚Äôt you captain save a hoe
558458967760830464,3,Oof
558459275618549802,3,Oof
558459502492647424,2,Hi
558459530900537375,28,<a:yoshi:553637790311776256>
558460057201803284,3,Rip
558460522274488320,25,<:Wut:519226309059149833>
558461102006992917,98,Does anyone else ever just get flashes of anger out of no where and want to take it out on someone
558461206361538580,2,Ok
558461260430180430,60,Well that‚Äôs happening rn and I want to ban the entire server
558461265165418526,3,Soo
558461288242610176,14,?ban @everyone
558461477330223114,96,<@!380928993144340482> my thigh hurts. I think I pulled it. How I pulled it tho is a funny story
558461633605926912,2,Ok
558461643298963467,3,DMs
558465461092876318,3,Wut
558465740030738462,3,Oof
558467056878485504,3,Rip
558467071080398868,3,Wtf
558468913915232257,11,Dasss a rip
558469091833282562,3,Oof
558469694290526244,76,If I mute Chelsea I expect she stays muted<a:eyesshaking:527915095892688896>
558469858174566422,6,No one
558469865023864832,5,CARES
558469956350771233,9,Yes I did
558469972381138947,20,And I‚Äôll do it again
558470008611536906,4,Jkjk
558470749480812554,1,O
558471035054194698,56,<@!449951831720853523> you lost vc privileges for a week
558471155619725322,49,‚ÄúThere‚Äôs a spic downstairs I‚Äôm gonna go kill him‚Äù
558471278520958986,42,You‚Äôre either dumb or ignorant of the word
558471441587240980,103,Spic is short for Hispanic it‚Äôs what gm white people called us. It‚Äôs like the n word for Spanish people
558471589331468289,4,Dies
558471807796248579,27,<@390924141697564675> Dewit
558471837034479617,5,Never
558472172381798433,3,Why
558472313083789322,26,?ban <@558457743761997825>
558472484031037440,82,You‚Äôre right <@!449951831720853523> I should‚Äôve banned but you‚Äôre lucky i like you
558472595159121931,9,Hennyways
558473926003523594,3,Oof
558757771990925320,3,Oof
558757922772090901,3,Oof
558760269296435200,3,Oof
558763574667771904,3,Wtf
558765143811555449,2,Oo
558766344397651968,33,<a:peekstitch:541444790349332510>
558772868071030811,3,Wut
558772882516213771,28,<a:yoshi:553637790311776256>
558772973209780225,9,Hennyways
558777697732395049,3,Oof
558781588700397588,3,Oof
558785599109332992,6,Lmfaoo
558785638237732865,10,I‚Äôm peeing
558787458431582209,12,Weeepwooooop
558787668163428353,31,<a:headtilt:486919048643215422>
558790368745750538,3,Oof
558797042768543765,2,Ok
558807799262347264,3,Oof
558812677879758869,5,Lmfao
558816365641990174,4,What
558820156030058496,4,Ooof
558820251597406208,42,<@556251957162016799> Whyd you say my name
558820588253347850,3,Oof
558822646058319872,14,I have too pee
558822846047059998,2,Ok
558823020186173461,3,Sad
558870918521290792,4,....
558871174965231626,125,?mute <@!449951831720853523> 60 that‚Äôs 2 weeks w/ no vc privileges since you wanted to find a way to be unmuted and undeafend
558871607679123456,32,?warn <@516951840550682625> spam
558872581474615338,3,Oof
559060697086951446,198,<@390924141697564675> it would‚Äôve been better if you handled this situation better and accepted that you weren‚Äôt picked this time instead of bitching about it. This just shows you‚Äôre to immatureü§∑‚Äç‚ôÇÔ∏è
559060805203656715,28,<@!398320585903702018> shhhh
560601986173763585,3,Oof
560620828887220224,25,You guys annoy me so much
560635277102809108,4,Poof
560652082869305376,3,Oof
560652332979847178,52,<:WTFW:543143882997760051> <:Wut:519226309059149833>
560652455851982849,36,<:PichuSurprised:552530445686800395>
560653141377548288,3,Oof
560653158393708545,4,K Gn
560653271644241981,56,When no one says good night back to me. What a great day
560975931536179202,3,Oof
560976003497721877,5,MuTeD
560976086494609409,1,K
560976238882193419,2,Kk
560976294473629723,26,?ban <@555864334048296964>
560976336202760228,4,YaWn
560976443962556426,84,You guys love aggravating me with your little shade throwing like I don‚Äôt realize it
560976601446219776,69,And when I say something ‚ÄúOmG sToP bEiNg such an ass to me and my gf‚Äù
560976696858116116,17,Hennyways I‚Äôm out
560976896888668171,113,<@431045920688177152> I was mimicking wat someone was saying please don‚Äôt get banned by not minding your business
560976961565097994,40,<@!285835383374348289> something to say?
560977000895086642,5,Oh ok
560977086836113428,4,Haii
560977154561802260,9,Who‚Äôs dad
560977186308489226,3,Oof
560977213458087956,18,No my dad is Dylan
560977222631161856,9,Try again
560977452331958272,25,<:Wut:519226309059149833>
560978861601456138,2,Ok
560978939623768084,1,K
560979088022568962,68,Maybe if things get really bad for you_ you can get back with hoodie
560979099011514399,3,OOF
560979125616246805,32,But things have to be really bad
560979228615507968,2,Ew
560979380147322883,30,He did the same to me at first
560979392743079936,7,For you
560979435092967436,30,You lose them how you get them
560979597722648599,79,Maybe we should just team up together and murder his entire family tree of dogs
560980110308802591,3,Oof
560981727619579943,7,Okizzle
560981804371410965,5,DELLA
560981839293186048,2,HƒØ
560981974492381184,39,I‚Äôve been in a bad mood for a long time
560981986601074700,7,Because
560982014497390622,25,I can‚Äôt take naps anymore
560982070977888276,4,Work
560982079655903252,4,Work
560982089273573416,10,And school
560982110635032616,14,Plus more work
560982155040129025,3,Yes
560982180474388520,8,Hospital
560984963344891904,3,YES
560985197789839391,3,Yay
561000433326227457,6,I peed
561000566365487124,13,Is it Sunday?
561000669620731915,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000691229786152,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000700981805056,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000707713400842,90,<@!325122999789551616> Once...Again do not bring private in a vc until the end of the week
561000817876795392,66,I mean I‚Äôve said it so many times but people continue to ignore me
561002875644280842,5,Dewit
561002890706026496,10,Then leave
561002898251579392,3,Lol
561003452319268896,13,I have to pee
561003504332832779,5,Dewit
561003553603321878,34,<@!325122999789551616> wanna fight
561003565678592024,115,<a:yoshi:553637790311776256> <a:yoshi:553637790311776256> <a:yoshi:553637790311776256> <a:yoshi:553637790311776256>
561003604878688266,34,I‚Äôll have thicc yoshi eat your ass
561003659370954775,5,NOOOO
561003691461574657,23,I‚Äôll piss in your mouth
561003713066696734,3,Yes
561003726840791040,13,Golden shower
561003765466136586,23,He doesn‚Äôt have to know
561003809002749954,14,But he doesn‚Äôt
561003839436881932,3,Not
