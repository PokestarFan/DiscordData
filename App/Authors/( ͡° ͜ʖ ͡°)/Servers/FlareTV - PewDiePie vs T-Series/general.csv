ID,Content Length,Content
559106126273183764,18,My Berryâ€™s are red
