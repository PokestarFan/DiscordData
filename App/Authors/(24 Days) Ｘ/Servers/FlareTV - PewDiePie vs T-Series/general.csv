ID,Content Length,Content
560977447558840361,18,I think it's over.
560977581340491787,43,T series literally just took over PewDiePie
