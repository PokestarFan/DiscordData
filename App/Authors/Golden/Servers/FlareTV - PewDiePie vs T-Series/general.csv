ID,Content Length,Content
560653879788830732,7,Remebet
560653914249101332,10,We can win
